pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -1249,13 +1249,10 @@ def process_circuits(  # noqa: PLR0912, PLR0915
                     quantinuum_circ = circuit_to_qasm_str(
                         c0,
                         header="hqslib1",
-                        maxwidth=(
-                            self.backend_info.misc.get(
-                                "max_classical_register_width", 32
-                            )
-                            if self.backend_info
-                            else 32
-                        ),
+                        # Override the reported value of max_classical_register_width
+                        # (which in August 2025 is 63 for all devices), to enable RNG
+                        # seeding which requires a 64-bit register.
+                        maxwidth=64,
```

Comment:
Do we want to put this in the changelog? Is this something that could potentially cause issues later on?
What happens if the seed register is used later on for something different?

Reply:
Probably worth putting in the changelog yes. I'll check that we can do other things with the seed register, and if not add a warning.